app/models/form/lettings/subsections/property_information.rb

```diff
@@ -3,7 +3,7 @@ def initialize(id, hsh, section)
     super
     @id = "property_information"
     @label = "Property information"
-    @depends_on = [{ "non_location_setup_questions_completed?" => true }]
+    @depends_on = [{ "non_location_setup_questions_completed?" => true, "supported_housing_and_renewal?" => false }]
```

Comment:
```suggestion
    @depends_on = [{ "non_location_setup_questions_completed?" => true, "supported_housing?" => false, "renewal?" => false }]
```

Remove the method — just be explicit.

Reply:
It should show if either supported housing is false or if renewal is false, not only if both are false

Reply:
You can stack them for `OR` statements:

```ruby
@depends_on = [
  { "non_location_setup_questions_completed?" => true, "is_supported_housing?" => false },
  { "non_location_setup_questions_completed" => true, "is_renewal?" => false }
]
```

Reply:
yes I forgot about this.
I will change to this for now until I discuss with Kat whether this should be here at all